src/applications/vaos/services/appointment/transformers.js

```diff
@@ -358,159 +358,48 @@ function setLocation(appt) {
-      return createPatientResourceFromRequest(appt);
-    }
-    case APPOINTMENT_TYPES.ccRequest: {
-      let contained = createPatientResourceFromRequest(appt);
-      if (appt.ccAppointmentRequest?.preferredProviders) {
-        appt.ccAppointmentRequest.preferredProviders.forEach(
-          (provider, index) => {
-            const address = [];
-            if (provider.address) {
-              address.push({
-                line: [provider.address?.street],
-                city: provider.address?.city,
-                state: provider.address?.state,
-                postalCode: provider.address?.zipCode,
-              });
-            }
-
-            contained = {
-              ...contained,
-              practitioner: {
-                id: `cc-practitioner-${appt.id}-${index}`,
-                name: provider.lastName
-                  ? {
-                      text: `${provider.firstName} ${provider.lastName}`,
-                      family: provider.lastName,
-                      given: provider.firstName,
-                    }
-                  : null,
-                address: provider.address ? address : null,
-                practitionerRole: [
-                  {
-                    location: [
-                      {
-                        reference: `Location/cc-location-${appt.id}-${index}`,
-                        display: provider.practiceName,
-                      },
-                    ],
-                  },
-                ],
-              },
-            };
-          },
-        );
-      }
-      return contained;
-    }
-    case APPOINTMENT_TYPES.ccAppointment: {
-      let address;
-      if (appt.address) {
-        address = {
-          line: [appt.address.street],
-          city: appt.address.city,
-          state: appt.address.state,
-          postalCode: appt.address.zipCode,
-        };
-      }
-
       return {
-        practitioner: {
-          resourceType: 'Location',
-          id: `cc-location-id`,
-          name: appt.providerPractice,
-          address: appt.address ? address : null,
-          telecom: appt.providerPhone
-            ? [
-                {
-                  system: 'phone',
-                  value: appt.providerPhone,
-                },
-              ]
-            : null,
-        },
+        vistaId: appt.facility?.facilityCode?.substring(0, 3),
+        stationId: appt.facility?.facilityCode,
       };
     }
-    case APPOINTMENT_TYPES.vaAppointment:
     default:
-      return null;
+      return {};
   }
 }
 
 /**
- * Returns an object containing data we may need from legacy var
+ * Returns contact information from a VAR request
  *
- * @param {Object} appt  VAR appointment object
- * @returns {Object}
+ * @param {VARRequest} appt  VAR appointment object
+ * @returns {PatientContact} An object containing the phone email the patient used in the request
  */
-function setLegacyVAR(appt) {
+function setContact(appt) {
```

Comment:
The field with this data in the VAOS service is called `contact`, so I named it that.